storage/miner.go

```diff
@@ -45,6 +46,17 @@ type Miner struct {
 
 	getSealConfig dtypes.GetSealingConfigFunc
 	sealing       *sealing.Sealing
+
+	sealingEvtType journal.EventType
+}
+
+// SealingStateEvt is a journal event that records a sector state transition.
+type SealingStateEvt struct {
```

Comment:
Ideally this would live inside the `sealing` package. We want to move `sealing.New` construction to DI, but it's somewhat entangled with the `Miner` struct here. We should try to not make this worse